src/domains/Auth/api/api.ts

```diff
@@ -102,11 +97,27 @@ export async function verifyEmail(userId: string) {
     return axios.post('/api/confirm/user-email', { userId });
 }
 
+export async function logout() {
+    return axios.post('/api/users/logout');
+}
+
 export async function getMyInfo() {
     // gets user info from the token in header
     return axios.get<ClientSafeUser>('/api/users/me');
 }
 
-export async function logout() {
-    return axios.post('/api/users/logout');
+export async function changeName(fname: string, lname: string) {
+    return axios.put<{ user: ClientSafeUser }>(`/api/users/me/name/`, { fname, lname });
+}
+
+export async function changeEmail(email: string) {
+    const match = email.match(/(\w+\.*)*\w+@(\w+\.)+\w+/gi);
+    if (!match || match[0].length !== email.length) {
+        throw errors.invalidEmail();
+    }
+    return axios.put<{ user: ClientSafeUser }>(`/api/users/me/email/`, { email });
+}
+
+export async function changeNotif(enabled: boolean, types: string[]) {
+    return axios.put<{ user: ClientSafeUser }>(`/api/users/me/notif`, { enabled, types });
```

Comment:
Lets spell out notifications here

Reply:
Done, 
right click on VSCode, clicked `Rename Symbol` and that looks like it renamed it everywhere

Reply:
Oh, I meant in the url too

Reply:
Done, also fixed spacing/formatting changes in other files